node/blockchain/tests/test_rest_api/test_signed_change_request/test_node_declaration.py

```diff
@@ -19,10 +19,30 @@ def test_node_declaration_signed_change_request_can_be_sent_via_api(api_client,
     assert signed_change_request.signature
 
     payload = signed_change_request.dict()
-    # TODO(dmu) CRITICAL: Expect `assert response.status_code == 204` instead once `perform_create()` is implemented
-    #                     https://thenewboston.atlassian.net/browse/BC-167
-    with pytest.raises(NotImplementedError, match=f'"identifier": "{regular_node.identifier}"'):
-        api_client.post('/api/signed-change-request/', payload)
+    response = api_client.post('/api/signed-change-request/', payload)
+    assert response.status_code == 201
+    assert response.json() == payload
+
+
+@pytest.mark.django_db
+def test_node_declaration_signed_change_request_with_invalid_account_lock(
+    api_client, regular_node, regular_node_key_pair
+):
+    message = NodeDeclarationSignedChangeRequestMessage(
+        node=regular_node,
+        account_lock='0' * 64,
+    )
+
+    signed_change_request = SignedChangeRequest.create_from_signed_change_request_message(
+        message=message,
+        signing_key=regular_node_key_pair.private,
+    )
+
+    payload = signed_change_request.dict()
+
+    response = api_client.post('/api/signed-change-request/', payload)
+    assert response.status_code == 400
+    assert response.json() == {'non_field_errors': ["['Invalid account lock']"]}
```

Comment:
something wrong with error format. It must be:
```
{'non_field_errors': ["Invalid account lock"]}
```

Either fix or put a TODO with a link to Jira task

Reply:
Fixed.